sources/packages/forms/src/form-definitions/programinformationrequest.json

```diff
@@ -1606,12 +1606,12 @@
             "tags": [],
             "properties": {},
             "conditional": {
-                "show": true,
-                "when": "pirStatus",
-                "eq": "Required",
+                "show": null,
+                "when": null,
+                "eq": "",
                 "json": ""
             },
-            "customConditional": "",
+            "customConditional": "show = !data.denyProgramInformationRequest && data.pirStatus !== \"Completed\" &&\n data.pirStatus !== \"Declined\"",
```

Comment:
This could be simplified as data.pirStatus === 'Required'. Am i missing anything?

Reply:
No you are not missing anything, I will update it.